nexus/tests/CMakeLists.txt

```diff
@@ -13,7 +13,7 @@ if(ADD_TEST)
   foreach(TESTNAME ${TESTLIST})
     #message("Adding test ntest_nexus_${TESTNAME}")
     set(NTEST "${qmcpack_SOURCE_DIR}/nexus/bin/nxs-test")
-    add_test(NAME ntest_nexus_${TESTNAME} COMMAND ${NTEST} -R ${TESTNAME} --ctest
```

Comment:
What is the added `\$` needed for here?

Reply:
It matches the end of the string. For instance, if `-R qdens` is specified, then qdens and qdens-radial will match, however, if `-R qdens$` is specified then only qdens will match. When there are multiple matches then the tests will not pass -- see for instance cecc6d9

Reply:
Here is the link to the failed test
https://github.com/QMCPACK/qmcpack/pull/3676/checks?sha=cecc6d9ec198bbf67dcc7886c75d018dd0e66def

Reply:
Error message placed here for convenience
```
Exactly one test should be selected when using ctest
    tests requested: ['qdens', 'qdens_radial']
```

Reply:
Yes, that looks like the behavior we want.  Thanks!